packages/fxa-auth-server/lib/db.js

```diff
@@ -761,7 +761,7 @@ module.exports = (
                       return true;
                     }
 
-                    if (sessionTokenId && device.sessionToken === sessionTokenId ||
+                    if (sessionTokenId && device.sessionTokenId === sessionTokenId ||
```

Comment:
Was this to address a prior bug of some sort?

Reply:
Oh, I meant to leave a comment about this - the field returned from the db is `sessionTokenId` but this was quietly changing it to `sessionToken` which lead to some frustrating test failure debugging. I changed to be `sessionTokenId` everywhere for consistency. If only we had some sort of technology that could prevent this kind of confusion in the first place... ;-)